source/options.html

```diff
@@ -19,7 +19,7 @@
 			name="token"
 			spellcheck="false"
 			autocomplete="off"
-			pattern="[\da-f]{40}"
+			pattern="([\da-f]{40})|(gp1_\w{36,251})"
```

Comment:
What is the grouping for? AFAIK you can use `|` in regex without having to be inside a group.

Reply:
I added them to try to organize the regex, but now I realize they are not necessary and might add unneeded overhead.  I will remove them.